docs/maui/essentials/folder-picker.md

```diff
@@ -0,0 +1,70 @@
+---
+title: FolderPicker - .NET MAUI Community Toolkit
+author: VladislavAntonyuk
+description: The FolderPicker provides the ability to pick a folder from the file system.
+ms.date: 12/11/2022
+---
+
+# FolderPicker
+
+The `FolderPicker` provides the ability to pick a folder from the file system.
+
+The following preconditions required for the `FolderPicker`:
+# [Android](#tab/android)
```

Comment:
As a general rule of thumb so we want to include all supported platforms and just add a 'nothing to do here' section for the ones that don't need anything? Or stick with no tab for those platforms?